### PR TITLE
fix: Improve responsiveness of event wizard

### LINKED ISSUE #6146: Need Adjustment in elements while seeing in mobile view
Screenshot (111)
**Describe the bug**

There should be a gap between these three buttons or need proper alignment

**Screenshots**

### PR DESCRIPTION
**Pull request by himanshujaidka**
Provides proper allignment to buttons in mobile view

Fixes #6146 

#### Short description of what this resolves:
Provides proper alignment to button in the mobile view

#### Checklist

- [✓ ] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [✓ ] My branch is up-to-date with the Upstream `development` branch.
- [✓ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [✓ ] I have added tests that prove my fix is effective or that my feature works
- [✓] I have added necessary documentation (if appropriate)

Screenshot (112)

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Use classes

File: app/components/forms/wizard/wizard-footer.hbs [hunk 1]
```diff
@@ -5,7 +5,7 @@
         {{t 'Cancel'}} <i class="close icon"></i>
       </LinkTo>
     {{else}}
-      <LinkTo @route="events.list" @model="live" class="ui orange button right labeled icon small {{if this.isUserUnverified 'disabled'}}">
+      <LinkTo @route="events.list" @model="live" class="ui orange button right labeled icon small {{if this.isUserUnverified 'disabled'}}" style="margin-top: 10px;">
         {{t 'Discard'}} <i class="close icon"></i>
```
Comment (afzal442) [inline app/components/forms/wizard/wizard-footer.hbs:9]: use mt class wherever required

**Review by afzal442 — COMMENTED** [review]
use mt class wherever required!

**Comment by himanshujaidka** [comment]
> Use classes

okay

**Comment by himanshujaidka** [comment]
> use mt class wherever required!

okay

**Comment by himanshujaidka** [comment]
@iamareebjamal sir, Made it, kindly review it

**Comment by himanshujaidka** [comment]
@iamareebjamal Thanks Sir

**Pull request merged**